modules/data_release/test/TestPlan.md

```diff
@@ -13,8 +15,8 @@ Ensure that regular users can see the data release module without the
  
 ## Permissions
 
-1. Give the user the `data_release_edit_file_access` *(Data Release: Give user 
-permission to view files)* permission.
+1. Give the user the `data_release_edit_file_access` *(Data Release: Grant other 
```

Comment:
I think the language here should match the README and give the proper name of the permissions.

I think it's `data_release_permissions` which is on a per-file basis? But I'm not sure. I'm not very familiar with the module.

Reply:
actually, this is the correct permission name. `data_release_permissions` is the table where the user permissions for data releases are stored.